Cargo.toml

```diff
@@ -9,6 +9,7 @@ members = [
     #    "sqlx-bench",
     "sqlx-mysql",
     "sqlx-postgres",
+    "sqlx-aws",
```

Comment:
As per the PR overview, I was originally going to build this crate out-of-tree (building on the new functionality), but implemented it in-tree as a way of verifying the new functionality was actually useful. After doing all that work, it now seems like leaving it in-tree is a reasonable idea. Here are the pros and cons I thought of:

pros:
- it's easy to align versions in-tree because of the Cargo workspace
- it makes the ecosystem of sqlx-* crates more easy to grok

cons:
- automated testing would require AWS credentials in this repo
- you (maintainers) might not want to maintain this